bin/lib/create.js

```diff
@@ -259,41 +273,53 @@ exports.create = function(project_path, config, options, events) {
 
         setShellFatal(true, function() {
             var project_template_dir = options.customTemplate || path.join(ROOT, 'bin', 'templates', 'project');
+            var app_path = path.join(project_path, 'app', 'src', 'main');
```

Comment:
Is this default? do you plan on having a way to create old project structure still? or is the plan to just support managing the old&existing structures while only being able to create the new (Android Studio) structure?